app/assets/stylesheets/events.scss

```diff
@@ -0,0 +1,208 @@
+// Place all the styles related to the events controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.add-event-form {
+  display: flex;
+  justify-content: center;
+  border-radius: 18px;
+  margin: 0;
+  width: 56.99%;
+  padding: 0;
+  position: absolute;
+  top: 35%;
+  left: 21.5%;
+  background-color: rgba(255, 255, 255, 0.75);
+}
+.events-header {
+  width: 60%;
+  margin: 15px;
+  width: 100%;
+  text-align: center;
+  color: #510280;
+}
+#col_en{
+  width: 50%;
+  margin-left: -11%;
+}
+#col_event_name{
+  width: 50%;
+  margin-left: 0%;
+}
+#event_name label{
+  width: 119%;
+  margin-top: 1.1%;
```

Comment:
I see a lot of different percentages values used for sizes and positioning. I'm afraid that so many values may be difficult to maintain and implement responsiveness for mobile devices in future if needed. Try to utilize bootstrap layouts for this job and reduce custom styles to minimum in future.